Tests/Builder/AbstractFormContractorTest.php

```diff
@@ -0,0 +1,177 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Tests\Builder;
+
+/**
+ * @author ju1ius <ju1ius@laposte.net>
+ */
+class AbstractFormContractorTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @param string $formType
+     * @param string $targetEntity
+     * @param string $associationType
+     * @param string $associationAdmin
+     * @param string $expectedException
+     *
+     * @dataProvider testFieldDescriptionValidationProvider
+     */
+    public function testFieldDescriptionValidation(
+        $formType,
+        $targetEntity,
+        $associationType,
+        $associationAdmin,
+        $expectedException
+    ) {
```

Comment:
> Sorry, but I don't want to be associated with that kind of crime against humanity practice.

Maybe, but this is not the place to do your fight. Let's focus on PR stuff and earn time. :+1: